lib/insights_cloud/async/insights_generate_notifications.rb

```diff
@@ -53,6 +53,10 @@ def add_notification(hits_count)
           }
         )
       end
+
+      def rescue_strategy_for_self
+        Dynflow::Action::Rescue::Skip
+      end
```

Comment:
Similar situation between `InsightsGenerateNotifications` and `InsightsRulesSync` as between `InventoryHostsSync`  and `InventorySelfHostSync`